src/main/java/yevhenii/lostfilmdemo/repository/SchedulerRequestRepository.java

```diff
@@ -0,0 +1,14 @@
+package yevhenii.lostfilmdemo.repository;
+
+import org.springframework.transaction.annotation.Transactional;
+import yevhenii.lostfilmdemo.entity.TVSeries;
+
+@Transactional
```

Comment:
don't put such annotations on interface level, it should be in implementation, in your case all implementations will be transactional which not always the requirenment